.gitlab-ci.yml

```diff
@@ -110,9 +102,6 @@ default:
   image: "${CI_IMAGE}"
   before_script:
     - !reference [.common-before-script, before_script]
-    - !reference [.prepare-env, before_script]
-    - !reference [.rust-info-script, script]
```

Comment:
why are you dropping these two?

Reply:
Because not all images used in linux-docker got rust